react/components/StickyRow.tsx

```diff
@@ -14,16 +20,27 @@ const CSS_HANDLES = ['headerStickyRow'] as const
 const StickyRow: FunctionComponent<Props> = ({ children, sticky, zIndex }) => {
   const handles = useCssHandles(CSS_HANDLES)
   const { offset, onResize } = useContext(RowContext)
+  const { scroll } = useScrollOffset()
```

Comment:
If row is not sticky (should it be a stickyRow?) we don't need to use this hook.